Typescript_TypeORM_Project/src/myRequest.d.ts

```diff
@@ -0,0 +1,19 @@
+// import { Request} from "express";
+declare namespace Express {
+  interface Request {
+    userId?: string;
+    userRole?: string;
+  }
+}
+// declare module "express" {
```

Comment:
aviod pushing comments which is not required

Reply:
updated